coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -1,3 +1,12 @@
+"""Functions that extract doc comments.
+
+The only public function extract_documentation_with_markers() is used by
+DocBaseClass.extract(). All it does is going over the code line by line
+and calling _extract_doc_comment_from_line(), which calls
+_extract_doc_comment(), which delegates extracting docs to three more
+specific functions: _extract_doc_comment_simple(),
+_extract_doc_comment_continuous() and _extract_doc_comment_standard().
+"""
```

Comment:
Hey although this is partially correct, I think it misses the point of the functions and how they integrate together.

How about something like this:

```
"""
Functions that extract documentation comments independently of any langage or documentation style.

Each of these functions is built upon one another, and at the last, exposes a single function `extract_documentation_with_markers` which is used by `DocBaseClass`, to extract documentation.
"""
```